.github/workflows/release-cli.yml

```diff
@@ -86,9 +93,151 @@ jobs:
+        include:
+          - GOOS: linux
+            GOARCH: amd64
+            pkgArch: amd64
+          - GOOS: linux
+            GOARCH: arm64
+            pkgArch: arm64
+    runs-on: ubuntu-24.04
+    steps:
+      - name: Get tag name
+        uses: olegtarasov/get-tag@2.1.3
+        id: tagName
+        with:
+          tagRegex: "cli/v(.*)"
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 22
+      - uses: pnpm/action-setup@v3
+        with:
+          version: 9
+      - uses: actions/setup-go@v5
+        with:
+          go-version: "1.23.4"
+          cache: false
+      - name: Run build script
+        run: make build VERSION=${{ steps.tagName.outputs.tag }}
+
+      - name: create dir structure for .deb package
+        run: |
+          mkdir -p .debpkg/usr/bin
+          cp -p bin/* .debpkg/usr/bin
+
+      - name: create a .deb package
+        uses: jiro4989/build-deb-action@v4
+        with:
+          package: kubetail
+          package_root: .debpkg
+          maintainer: Andres Amorey
+          version: ${{ steps.tagName.outputs.tag }}
+          arch: ${{ matrix.pkgArch }}
+          desc: cli tool to get realtime logs from kubernetes
+
+      - name: Calculate sha256 checksum
+        run: |
+          FILE='./kubetail_${{ steps.tagName.outputs.tag }}_${{ matrix.pkgArch }}.deb'
+          OUTPUT_FILE=${FILE}.sha256
+          sha256sum "$FILE" | cut -d " " -f 1 >"$OUTPUT_FILE"
+          mkdir -p deb-output
+          mv ./kubetail* ./deb-output/
+
+      - name: Upload .deb artifacts
+        uses: actions/upload-artifact@v4
+        with:
+          name: artifacts-${{ matrix.pkgArch }}-deb
+          path: deb-output/*
+
+  package-rpm:
+    name: Build rpm package
+    strategy:
+      matrix:
+        os: [ubuntu-24.04, ubuntu-24.04-arm]
+        include:
+          - os: ubuntu-24.04
+            GOOS: linux
+            GOARCH: amd64
+            pkgArch: x86_64
+          - os: ubuntu-24.04-arm
+            GOOS: linux
+            GOARCH: arm64
+            pkgArch: aarch64
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Get tag name
+        uses: olegtarasov/get-tag@2.1.3
+        id: tagName
+        with:
+          tagRegex: "cli/v(.*)"
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 22
+      - uses: pnpm/action-setup@v3
+        with:
+          version: 9
+      - uses: actions/setup-go@v5
+        with:
+          go-version: "1.23.4"
+          cache: false
+      - name: Run build script
+        run: make build VERSION=${{ steps.tagName.outputs.tag }}
+
+      - name: install the rpm and rpm build
+        run: sudo apt install rpm build-essential
+
+      - name: copy spec file to SPEC dir
+        run: |
+          mkdir -p ~/rpmbuild/{BUILD,RPMS,SOURCES,SPECS,SRPMS}
+          cp .github/workflows/kubetail-rpm.spec ~/rpmbuild/SPECS/kubetail.spec
```

Comment:
Let's put the .spec file in a new directory at `.github/ci-config`

Reply:
alright. we could use this for manifests we'll add for winget release ci as well.

Reply:
Sounds good